airflow/www/views.py

```diff
@@ -1895,6 +1895,20 @@ def trigger(self, session=None):
             flash(f"Cannot create dagruns because the dag {dag_id} has import errors", "error")
             return redirect(origin)
 
+        recent_confs = (
+            session.query(DagRun.conf)
+            .filter(
+                DagRun.dag_id == dag_id,
+                DagRun.run_type == DagRunType.MANUAL,
+                DagRun.conf.isnot(None),
+            )
+            .order_by(DagRun.execution_date.desc())
+            .limit(5)
+            .all()
```

Comment:
Can we change this to the first 5 unique configs?
If we can now select old configs, a lot of the options would just be the same.

Reply:
Whoops, yep! Somehow that slipped my mind.

Reply:
I'm a bit of an SQLalchemy noob--I thought that dropping in a .distinct after (or before) the .order_by clause would produce what we needed, but it doesn't seem to be working and I'm not sure why. Could I get a bit of guidance on this?

Reply:
I've tried this and it did not. Removing the .order_by made it work though.